### PR TITLE
Support hiding the map key

### PR DESCRIPTION
**Pull request by julienrf**
Add a new setting allowing the users to hide the map key. This feature was requested on the “Soaringemeteo v2 feedback” channel.

### CODE REVIEW

File: frontend/messages/sk.json [hunk 1]
```diff
@@ -31,6 +31,8 @@
     "settingsTimeUserTimezoneLegend": "Zobraziť čas pomocou môjho aktuálneho časového pásma namiesto UTC",
     "settingsTimeUTC": "Zobraziť čas podľa UTC",
     "settingsTimeUTCLegend": "Zobraziť čas UTC namiesto použitia môjho aktuálneho časového pásma",
+    "settingsMapKey": "Legendu",
+    "settingsShowMapKey": "Zobraziť legendu mapy",
```
[the remark this answers is not in the document]
Comment (slapiar) [reply frontend/messages/sk.json:35]: in is right: "settingsMapKey": "Legenda",
Other I confirm.

File: frontend/messages/pl.json [hunk 2]
```diff
@@ -31,6 +31,8 @@
 	"settingsTimeUserTimezoneLegend": "Pokaż czas, używając mojej strefy czasowej zamiast UTC",
 	"settingsTimeUTC": "Pokaż czas według UTC",
 	"settingsTimeUTCLegend": "Pokaż czas UTC zamiast mojej strefy czasowej",
+	"settingsMapKey": "Legendę",
+	"settingsShowMapKey": "Pokaż legendę mapy",
```
[the remark this answers is not in the document]
Comment (ant2753) [reply frontend/messages/pl.json:35]: settingsShowMapKey  OK
"settingsMapKey": "Legenda",

[the remark this answers is not in the document]
Comment (julienrf) [reply frontend/messages/pl.json:35]: Thank you, this is fixed!

**Comment by slapiar** [comment]
Vážený pán Skovajsa,

srdečná vďaka za veľký kus zdravia, ktoré pre nás vytvárate.
Príjemné prežitie sviatkov mieru a pokoja,

praje

Slavomír Piar

po 2. 12. 2024 o 19:04 Julien Richard-Foy ***@***.***>
napísal(a):

**Pull request merged**